### DISCUSSION TITLE
The user tag is ‘management‘ cannot log in to the UI

### DISCUSSION THREAD

**Discussion by yjqphp** [rabbitmq/rabbitmq-server#4878] [Other]
The user whose user tag is Management cannot log in to the UI

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
We cannot suggest much with a single line problem definition. At the very least we need

 * RabbitMQ and Erlang versions used
 * A set of specific steps to reproduce
 * Effective node configuration and `rabbitmq.conf` (see the Configuration guide to learn how to obtain effective node configuration)
 * Node logs

We do not guess in this community.
